record.go

```diff
@@ -29,7 +29,6 @@ type Record struct {
 	Notes                []string       `json:"notes,omitempty"`
 	Contents             []string       `json:"contents,omitempty"`
 	Summary              []string       `json:"summary,omitempty"`
-	Format               []string       `json:"format,omitempty"`
```

Comment:
Since this took a a slice of strings, I'm curious if we could instead just use this initial spec. I'm reluctant to move away from this as moving this data to the Holdings prevents us from being able to ever use facets / aggregations on Format. I believe storing multiple values here would allows us more flexibility in the future.

Or maybe it's a both thing as I can see it being useful on the specific Holding item as well?

Reply:
Yeah, we can definitely do both. I would strongly prefer that to just at the item level, since the  format is in fact specific to the manifestation. Will change today.